bootstrap-core/src/test/java/de/agilecoders/wicket/core/WicketApplicationTest.java

```diff
@@ -83,11 +83,11 @@ protected void init(WebApplication app) {}
     protected void onBefore() {
     }
 
-    protected Class<? extends Page> getHomePage() {
+    private Class<? extends Page> getHomePage() {
         return Page.class;
     }
 
-    protected IBootstrapSettings createBootstrapSettings() {
+    private IBootstrapSettings createBootstrapSettings() {
```

Comment:
those are `protected` intentionally. This way it is easier for a test to override whatever is needed.